src/jsx-dom.ts

```diff
@@ -272,6 +285,16 @@ function attribute(key: string, value: any, node: Element & HTMLOrSVGElement) {
         node.innerHTML = value["__html"]
       }
       return
+    case "value":
+      if (node instanceof window.HTMLTextAreaElement) {
+        node.value = value
+        return
+      } else if (node instanceof window.HTMLSelectElement) {
+        // apply value after creating `<option>` elements
```

Comment:
Aren’t `<option>` elements always created before the `<select>` due to the way JSX function call works?

Reply:
No, they are __created__ before parent node, but are __appended__ after attributes:
https://github.com/proteriax/jsx-dom/blob/35d58db68c783d3ab2f7a84abec90bc5f22a92a6/src/jsx-dom.ts#L146-L147